documentation/src/main/markdown/gettingstarted/terms/tools.md

```diff
@@ -1,12 +1,21 @@
-# [solution_name] tools
+# Tools
 
-[solution_name] tools are run to enable the scanning of your code.
+[solution_name] tools are components that enable the scanning of your source code. [solution_name] uses several underlying tools to perform scanning including:   
```

Comment:
Add container scanning?

Reply:
True story... 9.1.0 Container Scanning support. Will add that. Thank you.